src/main.ts

```diff
@@ -15,23 +15,37 @@ export class Main extends Command {
   }
 
   async run() {
+    if (this.argv.length === 0) return this._runDefaultCommand()
     let [id, ...argv] = this.argv
     this.parse({strict: false, '--': false, ...this.ctor as any})
     if (!this.config.findCommand(id)) {
       let topic = this.config.findTopic(id)
       if (topic) return this._help()
+      return this._runDefaultCommand()
     }
     await this.config.runCommand(id, argv)
   }
 
+  protected _getDefaultCommand(): string | undefined {
```

Comment:
Use getter syntax `protected get _defaultCommand ...`

Reply:
Thank you. This is now done.